api/middleware/logger.go

```diff
@@ -67,6 +65,9 @@ func Logger() macaron.Handler {
 		if len(ctx.Req.Form) > 0 {
 			paramsAsString += "?"
 			paramsAsString += ctx.Req.Form.Encode()
+		} else {
+			// requests that use POST with types like application/json will have the data in the body
+			//...which has already been read in by this point and we are totally screwed :/
```

Comment:
this comment is not really clear. we are screwed why? why is it a problem the body has been read? what are we trying to achieve here?

Reply:
it can probably be achieved with some hackery but i'm okay with keeping the limitation (and improving the comment)